x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -375,22 +375,27 @@ export enum HostStatus {
    * Default state of the host when no host information is present or host information cannot
    * be retrieved. e.g. API error
    */
-  ERROR = 'error',
+  UNHEALTHY = 'unhealthy',
```

Comment:
this will make the status "Unhealthy" if for some reason we cannot gather the Host data from Fleet.  I think that's OK, I just wanted to call it out.